tests/llmcompressor/modifiers/quantization/test_base.py

```diff
@@ -95,12 +95,11 @@ def test_block_strategy_parsing(block_q_config_kwargs):
 def test_actorder_resolution(
     has_actorder, actorder, q_config_kwargs, expected_0, expected_1
 ):
-    if has_actorder:
-        modifier = GPTQModifier(**q_config_kwargs, actorder=actorder)
-    else:
-        modifier = GPTQModifier(**q_config_kwargs)
-
     with pytest.raises(ValueError) if expected_0 == "error" else nullcontext():
+        if has_actorder:
```

Comment:
because GPTQ now calls `.resolve_quantization_config` on init, this error occurs during instantiation instead of at line `resolved = modifier.resolve_quantization_config()`